apache-hbase-mixin/signals/cluster.libsonnet

```diff
@@ -0,0 +1,316 @@
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_num_open_connections{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters',
+          },
+        },
+      },
+
+      regionServerConnections: {
+        name: 'RegionServer connections',
+        type: 'gauge',
+        description: 'Number of open connections to RegionServers.',
+        unit: 'short',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (region_server_num_open_connections{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - RegionServers',
+          },
+        },
+      },
+
+      // Authentication
+      masterAuthenticationSuccess: {
+        name: 'Master authentication successes',
+        type: 'raw',
+        description: 'Rate of successful authentications to the master.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_authentication_successes{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters success',
+          },
+        },
+      },
+
+      masterAuthenticationFailure: {
+        name: 'Master authentication failures',
+        type: 'raw',
+        description: 'Rate of failed authentications to the master.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (master_authentication_failures{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - masters failure',
+          },
+        },
+      },
+
+      regionServerAuthenticationSuccess: {
+        name: 'RegionServer authentication successes',
+        type: 'raw',
+        description: 'Rate of successful authentications to RegionServers.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (region_server_authentication_successes{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - rs success',
+          },
+        },
+      },
+
+      regionServerAuthenticationFailure: {
+        name: 'RegionServer authentication failures',
+        type: 'raw',
+        description: 'Rate of failed authentications to RegionServers.',
+        unit: 'reqps',
+        sources: {
+          prometheus: {
+            expr: 'sum by(' + groupAggList + ') (region_server_authentication_failures{%(queriesSelector)s})',
+            legendCustomTemplate: legendCustomTemplate + ' - rs failure',
+          },
+        },
+      },
+
+      // Master queue metrics
+      masterQueueSize: {
+        name: 'Master queue size',
+        type: 'gauge',
+        description: 'The size of the queue of requests, operations, and tasks to be processed by the master.',
+        unit: 'decbytes',
```

Comment:
If I am not mistaken, the master queue size refers to a number of items in the queue, which makes the `unit` a bit weird (decimal bytes), shouldn't be `short` or `none`? Since it's a number?